test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -1455,6 +1506,50 @@ def test_cgroup_cpuset(self):
                             'Processes should be assigned to different CPUs')
         self.logger.info('CpuIDs check passed')
 
+    def test_cgroup_cpuset_ncpus_are_cores(self):
+        """
+        Test to verify that correct number of jobs run on a hyperthread
+        enabled system when ncpus_are_cores is set to true.
+        """
+        # Check that system has hyperthreading enabled and has two processors
+        pcpus = 0
+        sibs = 0
+        cores = 0
+        with open('/proc/cpuinfo', 'r') as desc:
+            for line in desc:
+                if re.match('^processor', line):
+                    pcpus += 1
+                sibs_match = re.search(r'siblings	: ([1-9])', line)
+                cores_match = re.search(r'cpu cores	: ([1-9])', line)
+                if sibs_match:
+                    sibs = int(sibs_match.groups()[0])
+                if cores_match:
+                    cores = int(cores_match.groups()[0])
+        self.assertTrue(sibs > 0 and cores > 0)
```

Comment:
Should you also skip the test if sibs and cores are 0, rather than error it out?